Documentation/conf.py

```diff
@@ -0,0 +1,42 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = 'Autoscoper'
+copyright = '2023, Brown University and Kitware Inc'
```

Comment:
```suggestion
copyright = f'{date.today().year}, Brown University and Kitware, Inc.'
```

Also add `from datetime import date` at the top